app/assets/stylesheets/pages/about.scss

```diff
@@ -0,0 +1,108 @@
+.about-page  {
+  margin-top: 60px;
+  max-width: 1024px;
+  margin-bottom: 48px;
+
+  .about-intro {
+    &__title {
+      font-size: 18px;
+      height: 24px;
+      font-weight: bold;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.33;
+      letter-spacing: 0.3px;
+      color: $text-title;
+      margin-left: 6.5%;
+    }
+    &__paragraph {
+     margin-top: 16px;
+     height: 120px;
+     font-size: 18px;
+     font-weight: normal;
+     font-style: normal;
+     font-stretch: normal;
+     line-height: 1.33;
+     letter-spacing: normal;
+     color: $text-body;
+     letter-spacing: 0.13px;
+     margin-left: 6.5%;
+     margin-bottom: 20px;
+   }
+   &__line {
+     width: 452px;
+     height: 1px;
+     margin-left: 6.5%;
+     color: #bbbbbb;
+     background-color: #bbbbbb;
+     margin-top: .25%;
+   }
+  }
+
+  .team-members {
+    margin-top: .5%;
+    margin-left: 5%;
```

Comment:
This is something for later, but generally you want to avoid percentage margin and paddings and just go with media queries with sizes designated by the designer